checkov/terraform/checks/resource/BaseResourceBooleanValueCheck.py

```diff
@@ -0,0 +1,25 @@
+from abc import abstractmethod
+
+import dpath
+
+from checkov.terraform.models.enums import CheckResult
+
+from checkov.terraform.checks.resource.base_check import BaseResourceCheck
+
+expected_value = True
+
+
+class BaseResourceBooleanValueCheck(BaseResourceCheck):
+    def __init__(self, name, id, categories, supported_resources):
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf):
+        inspected_key = self.get_inspected_key()
+        if dpath.util.search(conf, inspected_key) != {}:
```

Comment:
can just write 
`if dpath.util.search(conf, inspected_key):`

Reply:
doesn't it mean "not null"?
not "empty dict" is different. no?

Reply:
No difference in that case